src/client_shared/conf/conf_cli_help.cpp

```diff
@@ -147,6 +147,19 @@ void PrintInTwoColumns(
 	}
 }
 
+string PrintArgument(CliArgument argument, ostream &stream) {
+	string cliarg;
+
+	if (!argument.name.empty()) {
```

Comment:
This works just fine.

A semantically better approach could be to use `optional` template, where in `CliCommand` we would instead have a `optional <CliArgument> argument;` attribute and in this line we would check with `argument.has_value()`. You can read more about this template [in here](https://github.com/mendersoftware/mender/blob/e8337ce91d9f6afec81fe7e62aa8a698c55726fa/src/common/optional.hpp) and search in our codebase for examples of usage.

Reply:
This sounds good, changed it to use this approach for both PR’s now 🙂